config/default.yml

```
@@ -2013,6 +2013,7 @@ Lint/RedundantDirGlobSort:
   Description: 'Checks for redundant `sort` method to `Dir.glob` and `Dir[]`.'
   Enabled: pending
   VersionAdded: '1.8'
```

Can you insert `VersionChanged: '<<next>>'`?
```suggestion
  VersionAdded: '1.8'
  VersionChanged: '<<next>>'
```

---

Updated, thank you